.github/workflows/publish_images_on_manual.yml

```diff
@@ -0,0 +1,32 @@
+name: On Demand (Manual) Docker Image Build
+
+on:
+  workflow_dispatch:
+    inputs:
+      trigger_type:
+        description: "Type of build which you want to trigger. Allowed values: main,tags,nightly"
```

Comment:
What does `main` mean exactly? Will it trigger builds on the default branch? The default branch is not `main` but `master` on most of our repos today.

Reply:
main means publishing of our main images: latest, alpine and postgres_12

Reply:
Several questions:
1. Will it actually use the default branch to create and update the packages?
2. Why do we want to publish those images manually? Why don't we publish those images only on pushes to `master` branch?

Reply:
1) Yes. There is a check in the code which prevent pushes from branches other than master. I haven't sent that PR yet
2) There is also one pipeline for pushes. I didn't add that pipeline since I can test that pipeline on my branch. Since I haven't completed my tests, I didn't send it to you. If you still want to check that code, the url for pipeline code is below
https://github.com/citusdata/docker/blob/update-version--docker-pipeline/.github/workflows/publish_images_on_push.yml

Reply:
For the time being, I am fine with manually triggering a workflow to create and push images for testing purposes.

However, I think we should remove this file in the long run, and allow image creation only after `git push` operations (build on all pushes and publish on commits&tags on default branch).

Reply:
We can always remove the manual workflow as you said. It  just requires deleting the pipeline